src/test/run-pass-fulldeps/create-dir-all-bare.rs

```diff
@@ -8,18 +8,12 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-// ignore-cross-compile
-
-#![feature(rustc_private)]
-
-extern crate tempdir;
-
 use std::env;
 use std::fs;
-use tempdir::TempDir;
+use std::path::PathBuf;
 
 fn main() {
-    let td = TempDir::new("create-dir-all-bare").unwrap();
-    env::set_current_dir(td.path()).unwrap();
+    let path = PathBuf::from(env::var_os("RUST_TEST_TMPDIR").unwrap());
+    env::set_current_dir(&path).unwrap();
```

Comment:
Why are these changes necessary? Can we no longer link to tempdir from tests? Or is it that tempdir wasn't being rebuilt because rustbuild is buggy (I would *not* be surprised).

Reply:
The dependencies of librustc_trans are no longer available in the sysroot (as we just manually copy the one DLL), and one of its dependencies was `tempdir`. (no other crates in rustc use `tempdir`)

Reply:
As I recall, the previous solution to this "problem" would be to move tempdir to a dependency of `librustc`. I'm fine with this too though.

Reply:
It's true yeah we could do that but I'm also always a fan of tests with fewer dependencies :)